### PR TITLE
Max - Test four

### PR DESCRIPTION
**Pull request by FireMugen**
Completed with css grid and without editing the index.html

### CODE REVIEW

File: test-four/main.css [hunk 1]
```diff
@@ -3,7 +3,34 @@ body {
...
 .container {
-  padding: 30px;
+  padding: 2.1vw;
+  margin: auto;
+  max-width: 1200px;
 }
 
-/* Place your CSS here... */
+.card-container {
+  display: grid;
```
Comment (danwfeeney) [inline test-four/main.css:12]: I like that your using CSS Grid, but in plain CSS I don't think this will not in IE?

File: test-four/main.css [hunk 2]
```diff
@@ -3,7 +3,34 @@ body {
...
+}
+
+.card__body {
+  padding: 15px;
+  height: 160px;
+  box-shadow: 0 0 3px lightgray;
+}
+
+.card__title {
+  font-size: 20;
```
Comment (danwfeeney) [inline test-four/main.css:30]: shouldn't a unit be explicitly specified here? https://stackoverflow.com/questions/7907749/default-unit-for-font-size

File: test-four/main.css [hunk 3]
```diff
@@ -3,7 +3,34 @@ body {
 }
 
 .container {
-  padding: 30px;
+  padding: 2.1vw;
```
Comment (danwfeeney) [inline test-four/main.css:6]: can you explain why you're using a mixture of view width, pixels and rems?

**Review by danwfeeney — COMMENTED** [review]
Solid result, but would like to have seen a properly responsive solution.

File: test-four/main.css (hunk 2 above)
Comment (FireMugen) [reply test-four/main.css:30]: Yes you are right I should have specified a unit here, I guess the size was inherited from somewhere.

File: test-four/main.css (hunk 1 above)
Comment (FireMugen) [reply test-four/main.css:12]: Yes in hindsight I should have used vendor prefixes to make my solution work in IE seeing as IE does have an implicit grid.

File: test-four/main.css (hunk 3 above)
Comment (FireMugen) [reply test-four/main.css:6]: > Solid result, but would like to have seen a properly responsive solution.

Thanks! Yes in hindsight I probably should have used media queries I just wanted to try something different (If thats what you mean by properly responsive?).

Comment (FireMugen) [reply test-four/main.css:6]: > can you explain why you're using a mixture of view width, pixels and rems?

I used view width here so when the window size changes the padding scales in a pleasing way. 

After some reading I can see that I should have used REMs for most things and used pixels a lot more sparingly(just for thin lines and some other measurements).

**Pull request closed without merging**